source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -129,6 +240,109 @@ This means that in order to have a VM connected to both ipv4 and ipv6 subnets,
 it needs to connect to 2 different networks (through 2 different logical ports),
 one with an IPv4, other with an IPv6 subnet defined on top.
 
+### Networking API entity provisioning
+
+Below you can find the steps required to have dynamic IPv6 addresses on the
+guest VMs, along with the available methods of provisioning it.
+
+1. create a logical network
+    - oVirt UI
+    - oVirt REST API
+    - manageIQ UI
+    - ovirt-provider-ovn REST API. [REST example](#create-a-logical-network)
+2. add a subnet on top
+    - oVirt UI - *requires changes*
+    - manageIQ UI
+    - ovirt-provider-ovn REST API. [REST example](#create-a-subnet)
+3. provision a router
+    - ovirt-provider-ovn REST API. [REST example](#create-a-router)
+4. attach the subnet to the router
+    - oVirt ovirt-provider-ovn REST API. [REST example](#configure-the-router)
+5. create a VM having the vNIC profile of the network created in step #1
+   **or** assign the vNIC profile of the network created in #1 to an existing
+   VM
+    - oVirt UI
+    - oVirt REST API
+
+#### EL7 Guest VM configuration
+
+For dhcpv6-stateless the guest VM can be configured using *either* of the 3
+configuration options defined in the [Guest config section](#guest-vm-configuration).
+
+It is important to state that if and when the dhclient version supplied in EL7
+stops defaulting to /64 prefixes, the guest VMs have to be configured as
+indicated for [EL8](#el8-guest-vm-configuration).
+
+#### EL8 Guest VM configuration
+
+On the version of dhclient supplied in el8, dhcpv6 alone is **not enough** to
+configure dynamic IP addresses on the guest VMs.
```

Comment:
Is NetworkManager of RHL8 using dhclient at all?

Reply:
Depends on the configuration ... 

From the man page of the NetworkManager.conf on the el8 on the template our QEs use (1.14.0-5.el8) , it is dhclient.

Never the less, from bugs [0], [1], and  [2] - along with knowledge article [3] - I think this is the overall direction of the dynamic IP configuration of the industry - configure network prefixes lengths of 128.

[0] - https://bugs.launchpad.net/ubuntu/+source/isc-dhcp/+bug/1609898
[1] - https://bugzilla.redhat.com/show_bug.cgi?id=1635181
[2] - https://bugzilla.redhat.com/show_bug.cgi?id=1673951
[3] - https://kb.isc.org/docs/aa-01141